.pre-commit-config.yaml

```
@@ -0,0 +1,18 @@
+fail_fast: true
+
+repos:
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+    -   id: black
+        name: "Black format checker (source)"
+        args: ["--check"]
+        files: ^src/
+        types: [file, python]
+        language: python
+    -   id: black
+        name: "Black format checker (tests)"
+        args: ["--check"]
+        files: ^tests/
+        types: [file, python]
+        language: python
```

```suggestion
    hooks:
    -   id: black
        language_version: python3
```

If checking both the `src` and `tests` why not just Black everything?

---

Yes, I am onboard with this (leaving the `--check` arg). 

The only reason to explicitly specify was to avoid `venv`, but that is handled by the `pyproject.toml` config :)